tests/app/main/views/test_services.py

```diff
@@ -90,12 +92,6 @@ def test_service_view_no_features_or_benefits_status_disabled(self, data_api_cli
             "normalize-space(string(//td[@class='summary-item-field']//*[@class='service-id']))"
         ) == "314159265"
 
-        assert frozenset(document.xpath("//input[@type='radio'][@name='service_status']/@value")) == frozenset((
```

Comment:
Are some of these tests duplicates now that the assertions about statuses have gone?  Feels like we can lose one or two of them altogether probably?